junebug/stores.py

```diff
@@ -89,7 +89,7 @@ def load_vumi_message(self, channel_id, message_id):
 class OutboundMessageStore(BaseStore):
     '''Stores the event url, in order to look it up when deciding where events
     should go'''
-    PROPERTY_KEYS = ['event_url']
+    PROPERTY_KEYS = ['event_url', 'event_auth_token']
```

Comment:
For consistency with `mo_url_auth_token` I would suggest we use `event_url_auth_token` even though it is quite verbose.